dolphinscheduler-api/src/main/resources/application.yaml

```diff
@@ -204,6 +207,20 @@ security:
           callbackUrl: ""
           iconUri: ""
           provider: google
+casdoor:
+  # Your Casdoor server url
+  endpoint: ""
+  client-id: ""
+  client-secret: ""
+  # The certificate may be multi-line, you can use `|-` for ease
+  certificate: ""
+  # Your organization name added in Casdoor
+  organization-name: ""
+  # Your application name added in Casdoor
+  application-name: ""
+  # Doplhinscheduler login url
+  redirect-url: ""
```

Comment:
I see the Java code change for casdoor user admin,and how about this config used?

Reply:
Similar to LDAP, determine the user type when creating a local user

Reply:
It's configuration about casdoor and we don't need to write code for this part?

Reply:
The casdoor function has been submitted before, but the configuration was not submitted. I just added the casdoor user admin parameter and submitted the configuration by the way.